compiler/damlc/daml-doc/test/DA/Daml/Doc/Tests.hs

```diff
@@ -49,12 +50,12 @@ mkTestTree externalAnchors = do
   expectFiles <- filter isExpectationFile <$> listDirectory testDir
 
   let goldenSrcs = nubOrd $ map (flip replaceExtensions "daml") expectFiles
-  goldenTests <- mapM (fileTest externalAnchors  . (testDir </>))  goldenSrcs
+  goldenTests <- mapM (fileTest externalAnchors scriptPackageData . (testDir </>)) goldenSrcs
 
-  pure $ Tasty.testGroup "DA.Daml.Doc" $ unitTests <> concat goldenTests
+  pure $ Tasty.testGroup "DA.Daml.Doc" $ unitTests scriptPackageData <> concat goldenTests
 
-unitTests :: [Tasty.TestTree]
-unitTests =
+unitTests :: ScriptPackageData -> [Tasty.TestTree]
```

Comment:
Small suggestion: unless these tests need access to daml-script, I'd remove this argument here and modify `damldocExpect`, `damldocExpectMany`, `runDamldoc`, etc to take it in a `Maybe`, as we do for `importPathM`.

Reply:
Oh right yeah, I had intended to add a Script test in those but that seemed to slip my mind. Think its worth adding there? Or is the integration test I added enough

Reply:
I settled on integration test, since I don't need to make assertions about the result, only that it works